Add confluent schema registry for Kafka sink Protobuf (#1982)
risingwavelabs/risingwave-docs: docs/guides/create-sink-kafka.md

|Field|Notes|
|-----|-----|
|message| Required. Message name of the main Message in the schema definition. . |

> |schema.registry| Required. The address of the schema registry. This can be in either `file://`, `http://`, `https://` format, or a Confluent schema registry. |

Reviewer: ```suggestion
|schema.location| The schema location. This can be in either `file://`, `http://`, `https://` format. |
|schema.registry| The address of the schema registry. |
```
Exactly one of `schema.location` or `schema.registry` shall be specified.
